packages/jellyfish-transaction-builder/src/txn/txn_builder_masternode.ts

```diff
@@ -41,4 +48,90 @@ export class TxnBuilderMasternode extends P2WPKHTxnBuilder {
       changeScript
     )
   }
+
+  /**
+   * Build update masternode transaction
+   *
+   * @param {UpdateMasternode} updateMasternode transaction to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @param {Object} [collateralInfo] needed when updating owner address
+   * @param {RawTransaction} collateralInfo.rawCollateralTx to get relevant data needed for collateral vin and vout
+   * @param {Script} collateralInfo.newOwnerScript for new owner address
+   * @return {Promise<TransactionSegWit>}
+   */
+  async update (
+    updateMasternode: UpdateMasternode,
+    changeScript: Script,
+    collateralInfo?: {
+      rawCollateralTx: RawTransaction
```

Comment:
As described above, we might need to use another more prevalent data type that is available everywhere. Since `RawTransaction` might have more information that we might not need. We need to keep the scope small; we could also declare the type here.

Reply:
ok, I replaced the raw transaction param with txid, value, tokenId